recipe/meta.yaml

```diff
@@ -37,8 +37,9 @@ outputs:
         - certifi >=2020.06.20
         - freetype >=2.3
         - numpy   1.19  # [not (osx and arm64) and (py==37 or py==38 or py==39)]
+        - numpy   1.19  # [(linux and ppc64le) and (py==37 or py==38 or py==39)]
```

Comment:
Out of curiosity, is this covered by the line above? Can they be combined?

Reply:
I tried to combine them but readability is getting worse

Reply:
@adipietro18 Oh, finally, I fixed it